docker/docker_update.sh

```diff
@@ -3,4 +3,4 @@ set -e
 
 mkdir -p volumes/pictrs
 sudo chown -R 991:991 volumes/pictrs
-sudo docker-compose up -d --build
+sudo docker compose up -d --build
```

Comment:
Looks like a mistake.

Reply:
Docker is now bundling compose , and this is the new preferred way anyway.